src/ucp/api/ucp.h

```diff
@@ -939,20 +947,36 @@ typedef struct ucp_generic_dt_ops {
  * @ingroup UCP_DATATYPE
  * @brief UCP datatype attributes
  *
- * This structure provides attributes that can be queried for a UCP datatype.
+ * This structure provides attributes of a UCP datatype.
```

Comment:
This structure provides attributes for querying a UCP datatype
@tonycurtis WDYT?

Reply:
agree